webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -171,18 +166,12 @@ <h1>Product Navigation</h1>
                     <a href="{{ url('manage:home') }}">Admin</a>
                     {% endif %}
                     |
-                    {% if request.session.get('signin_method') == 'google' %}
+                    <form class="logout-form" action="/oidc/logout/" method="POST">
```

Comment:
I missed mentioning this earlier, but we'll want to use the `url` helper for generating the URL here as well.

Reply:
The URL helper disappeared, but we still want to use it here.